fix(jans-auth-server): common-text upgrade 1.11.0 leads to class initialization problems

**Describe the bug**

bot raised PR to upgrade common-text 1.10.0 -> 1.11.0 which was merged into main. This leads to initialization failure for `org.apache.commons.text.StringEscapeUtils` because it depends on common-lang and 1.11.0 does not like our current version. 

https://github.com/JanssenProject/jans/pull/6743
```
Caused by: jakarta.el.ELException: //opt/jetty-11.0/temp/jetty-localhost-8081-jans-auth_war-_jans-auth-any-8676923540685489908/webapp/authorize.xhtml @29,73 value="#{authorizeAction.loginHint}": Error writing [login
Hint] on type [io.jans.as.server.authorize.ws.rs.AuthorizeAction]
        at com.sun.faces.facelets.el.TagValueExpression.setValue(TagValueExpression.java:101) ~[jakarta.faces-3.0.2.jar:3.0.2]
        at jakarta.faces.component.UIInput.updateModel(UIInput.java:834) ~[jakarta.faces-3.0.2.jar:3.0.2]
        ... 56 more
Caused by: jakarta.el.ELException: Error writing [loginHint] on type [io.jans.as.server.authorize.ws.rs.AuthorizeAction]
        at jakarta.el.BeanELResolver.setValue(BeanELResolver.java:117) ~[org.mortbay.jasper.apache-el-10.0.14.jar:10.0.14]
        at com.sun.faces.el.DemuxCompositeELResolver._setValue(DemuxCompositeELResolver.java:199) ~[jakarta.faces-3.0.2.jar:3.0.2]
        at com.sun.faces.el.DemuxCompositeELResolver.setValue(DemuxCompositeELResolver.java:222) ~[jakarta.faces-3.0.2.jar:3.0.2]
        at org.apache.el.parser.AstValue.setValue(AstValue.java:201) ~[org.mortbay.jasper.apache-el-10.0.14.jar:10.0.14]
        at org.apache.el.ValueExpressionImpl.setValue(ValueExpressionImpl.java:262) ~[org.mortbay.jasper.apache-el-10.0.14.jar:10.0.14]
        at org.jboss.weld.module.web.el.WeldValueExpression.setValue(WeldValueExpression.java:64) ~[weld-web-4.0.3.Final.jar:4.0.3.Final]
        at com.sun.faces.facelets.el.TagValueExpression.setValue(TagValueExpression.java:95) ~[jakarta.faces-3.0.2.jar:3.0.2]
        at jakarta.faces.component.UIInput.updateModel(UIInput.java:834) ~[jakarta.faces-3.0.2.jar:3.0.2]
        ... 56 more
Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.apache.commons.text.StringEscapeUtils
        at io.jans.as.server.authorize.ws.rs.AuthorizeAction.setLoginHint(AuthorizeAction.java:783) ~[classes/:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at jakarta.el.BeanELResolver.setValue(BeanELResolver.java:113) ~[org.mortbay.jasper.apache-el-10.0.14.jar:10.0.14]
        at com.sun.faces.el.DemuxCompositeELResolver._setValue(DemuxCompositeELResolver.java:199) ~[jakarta.faces-3.0.2.jar:3.0.2]
        at com.sun.faces.el.DemuxCompositeELResolver.setValue(DemuxCompositeELResolver.java:222) ~[jakarta.faces-3.0.2.jar:3.0.2]
        at org.apache.el.parser.AstValue.setValue(AstValue.java:201) ~[org.mortbay.jasper.apache-el-10.0.14.jar:10.0.14]
        at org.apache.el.ValueExpressionImpl.setValue(ValueExpressionImpl.java:262) ~[org.mortbay.jasper.apache-el-10.0.14.jar:10.0.14]
        at org.jboss.weld.module.web.el.WeldValueExpression.setValue(WeldValueExpression.java:64) ~[weld-web-4.0.3.Final.jar:4.0.3.Final]
        at com.sun.faces.facelets.el.TagValueExpression.setValue(TagValueExpression.java:95) ~[jakarta.faces-3.0.2.jar:3.0.2]
        at jakarta.faces.component.UIInput.updateModel(UIInput.java:834) ~[jakarta.faces-3.0.2.jar:3.0.2]
        ... 56 more
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.NoSuchMethodError: 'org.apache.commons.lang3.Range org.apache.commons.lang3.Range.of(java.lang.Comparable, java.lang.Comparable)' [in thread "qtp
842326585-24"]
        at org.apache.commons.text.translate.NumericEntityEscaper.<init>(NumericEntityEscaper.java:97) ~[commons-text-1.11.0.jar:1.11.0]

```


Snyk shows no vulnerability for 1.10.0 version, so no reason to upgrade. 
https://security.snyk.io/package/maven/org.apache.commons:commons-text

![image](https://github.com/JanssenProject/jans/assets/2013070/023bac7a-cf92-4653-95da-9fd79f27e2c2)

--- a/jans-bom/pom.xml
+++ b/jans-bom/pom.xml
@@ -482,7 +482,7 @@
 			<dependency>
 				<groupId>org.apache.commons</groupId>
 				<artifactId>commons-text</artifactId>
-				<version>1.11.0</version>
+				<version>1.10.0</version>
 			</dependency>
 			<dependency>
 				<groupId>commons-beanutils</groupId>

